### DISCUSSION TITLE
Is there a limited to how many hobby projects you can create?

### DISCUSSION THREAD

**Discussion by karimfromjordan** [sqlitecloud/discussions#25] [Dashboard] [answered]
I was expecting to only be able to create a single hobby project. But it looks you can create several, all for free and with their own usage limits. Is this expected?

**Comment by gionatamettifogo** [comment, marked as answer]
Hi Karim,

We don't currently enforce limits on hobby plans so please go ahead with fair use; if we can help, please let us know.

-- Gionata
